demos/rag_agentic/notebooks/Level4_agentic_and_mcp.ipynb

```diff
@@ -662,6 +662,15 @@
     "In the screenshot above, you can see that our `et-slack-bot` which is configured to both our public Slack workspace and the Slack MCP server, successfully posted a pod status summary message in the #demos channel."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "b0d0a538-96e0-4007-ad58-d0308f25d047",
+   "metadata": {},
+   "source": [
+    "### Demo Recording\n",
+    "You can also check out this recorded demo: https://youtu.be/aaxclldaVWw — it walks through the detailed steps for setting up a Slack app, configuring the Slack MCP server and running the query shown above."
```

Comment:
Can we add this to the `demos/rag_agentic/README.md` instead of the notebook?

Reply:
sure